File: hello.py
# Hello World — Python

from typing import Optional
import asyncio


def greet(name: str = "World") -> str:
    return f"Hello, {name}!"


names = ["Alice", "Bob", "Claude"]

for name in names:
    print(greet(name))


# Dataclass example
from dataclasses import dataclass

@dataclass
class Person:
    name: str
    age: int

    def introduce(self) -> str:
        return f"I'm {self.name}, {self.age} years old."


# Async example
async def fetch_greeting(url: str) -> Optional[str]:
    import urllib.request
    with urllib.request.urlopen(url) as resp:
        return resp.read().decode()


if __name__ == "__main__":
    p = Person("Claude", 1)
    print(p.introduce())

File: hello.js
// Hello World — JavaScript

function greet(name = "World") {
  return `Hello, ${name}!`;
}

const names = ["Alice", "Bob", "Claude"];

for (const name of names) {
  console.log(greet(name));
}

// Arrow function
const double = (x) => x * 2;
console.log(double(21)); // 42

// Async example
async function fetchGreeting(url) {
  const res = await fetch(url);
  const data = await res.json();
  return data.greeting ?? "Hello!";
}
